tests/foreman/ui/test_contentview.py

```diff
@@ -1035,6 +1037,125 @@ def test_positive_create_date_filter_rule_without_type(self):
             self.assertEqual(result['start_date'], start_date)
             self.assertEqual(result['end_date'], end_date)
 
+    @tier2
+    def test_positive_add_all_security_errata_date_range_filter(self):
```

Comment:
`test_positive_add_all_security_errata_by_date_range_filter`

Reply:
I was thinking wrong - `add all security errata to date range filter`, and `to` is usually omitted in titles. You're right, it should be `by` since we add errata to cvv, not filter.